packages/circuit-ui/package.json

```diff
@@ -56,7 +56,7 @@
     "@emotion/react": "^11.8.2",
     "@emotion/styled": "^11.8.1",
     "@sumup/collector": "^1.4.0",
-    "@sumup/design-tokens": "^3.4.0",
+    "@sumup/design-tokens": "^4.0.0-canary.0",
```

Comment:
I don't think we actually _have_ to force this to be >v4 but I would anyways—the upgrade should happen simultaneously and it'll make it easier for us _not_ to have to think about whether we're breaking `circuit-ui` for apps still on `design-tokens@3`